### PR TITLE
Branding api

### PR DESCRIPTION
**Pull request by C0rby**
New endpoints for the custom logo upload and reset.
The static files are served by the `web` service.

When the user uploads a logo the backend also updates the `theme.json` to include the new logo path.
The files are stored under the `WEB_ASSET_PATH` which defaults to `$OCIS_BASE_DATA_PATH/web/assets`.

### CODE REVIEW

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>wopiValidatorTests-ocis</strong>](https://drone.owncloud.com/owncloud/ocis/19049/40/1) failed. Further test are cancelled...

File: services/web/pkg/config/config.go [hunk 1]
```diff
@@ -22,7 +22,10 @@ type Config struct {
 	File  string `yaml:"file" env:"WEB_UI_CONFIG" desc:"Read the ownCloud Web configuration from this file."` // TODO: rename this to a more self explaining string
 	Web   Web    `yaml:"web"`
 
-	Context context.Context `yaml:"-"`
+	TokenManager *TokenManager `yaml:"token_manager"`
+
+	GatewayAddress string          `yaml:"gateway_addr" env:"WEB_GATEWAY_GRPC_ADDR" desc:"GRPC address of the Reva gateway service."`
```
Comment (mmattel) [inline services/web/pkg/config/config.go:27]:
```suggestion
	GatewayAddress string          `yaml:"gateway_addr" env:"WEB_GATEWAY_GRPC_ADDR" desc:"The bind address of the GRPC service."`
```
This is the description text we use all over when it comes to `xyz_GRPC_ADDR`

**Comment by mmattel** [comment]
@C0rby I know that there is no readme.md currently present and I am not asking for a full blown one - but would you mind adding one and just _dump_ down the changes you just made. We would then have at least a startingpoint and things wont be forgotten if we polish that readme with more content 😅 🙏

**Comment by C0rby** [comment]
@mmattel, I'll add the readme in another PR to not bloat this one. :+1:

**Review by micbar — APPROVED** [review]
LGTM

**Comment by C0rby** [comment]
> Can you check that the default value gets added as I have posted in the suggestable commit?

The default value is here: https://github.com/owncloud/ocis/pull/5559/files#diff-a1bba8d1ace694b6b8a7c869be805bbfd6b0a3b140738fecfadcfdcd4dd47bf9R38. The config doc generator will take it from there or am I missing something?

**Comment by C0rby** [comment]
The wopiserver fails to start for some reason... or rather it doesn't even seem to start it...
https://drone.owncloud.com/owncloud/ocis/19058/40/6

**Comment by mmattel** [comment]
> The default value is here

Sorry, I missed that 🙈

**Pull request merged**